### ISSUE TITLE
Unable to use JSON object data from custom API call in swipeable stack on swipe right/left action

### ISSUE THREAD

**Issue by stefanlenoach** [FlutterFlow/flutterflow-issues#439]
When creating a new document in Firebase, I expected to be able to access the JSON swipeable stack items similar to how I can in any of the UI elements (text via $.name for example). Instead, the UI doesn't allow me to make this simple data reference, if it's possible to do what I'm trying to do, it's very unintuitive. 

## To Reproduce 
1. Get a list of data objects from a custom API call
2. On the swipeable stack widget, connect it to the custom API call so that it will be populated by the items returns by it
3. On the action edit section, try to create a new document in the Firestore that uses some data ('name' for example) from the objects returned in the custom API call.

## Context
Basically just trying to do a simple create action on swipe left/right and it seems with custom API data the UI doesn't make this easy.

## Screenshots / recordings
https://www.loom.com/share/838175e375b94b6c97bd830a138e5df7

**Comment by Pepino-GR** [comment]
Hi stefanlenoach,

 it seems, that the data from the API call are not available inside swipe actions. So you can't use it inside the context. It is not a bug, because it works as intend. If you want to make a feature request, I can set it.

**Comment by stefanlenoach** [comment]
Yes please!! It makes sense that I should be available to do this, yes?

**Comment by shelpert** [comment]
Hello,

This is in fact a bug, thanks for reporting! The swipeable stack widget is a bit of a special case which is why this got overlooked. We have made a fix for this and you will see it in a future update!
